src/services/qrService.ts

```diff
@@ -380,7 +381,7 @@ const connectOpenId4VcIssuance = async (args: IQrDataArgs): Promise<void> => {
     }).then((contacts: Array<IContact>) => {
       if (contacts.length === 0) {
         args.navigation.navigate(ScreenRoutesEnum.CONTACT_ADD, {
-          name: url.hostname,
+          name,
```

Comment:
name || url.hostname

Reply:
I'm doing exactly that a few lines above 370

Reply:
Oh okay, because of the long line in Github didn't see that